src/communication-security/grpc-code/grpc_client_secured/client.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"context"
+	"crypto/tls"
+	"crypto/x509"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	pb "pentest/grpc/samplebuff"
+	"time"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+)
+
+const (
+	defaultName = "Art Rosenbaum"
+)
+
+var (
+	addr = flag.String("addr", "localhost:10001", "Address of Server")
+	name = flag.String("name", defaultName, "Name to greet")
+)
+
+func main() {
+	flag.Parse()
+	b, _ := ioutil.ReadFile("../cert/ca.cert")
```

Comment:
nit: `ioutil` package is deprecated since go1.16. In this case we should use `os.ReadFile` instead.

Reply:
nit: let's promote best practice of not skipping errors
```suggestion
	b, err := ioutil.ReadFile("../cert/ca.cert")
	if err != nil {
		log.Fatalf("Could read ca.cert: %v", err)
	}
```

Reply:
@anatolym Shouldn't the error message be "Couldn't read ca.cert: %v", instead?